src/QsCompiler/Tests.LanguageServer/ProjectLoaderTests.cs

```diff
@@ -15,23 +14,8 @@ namespace Microsoft.Quantum.QsLanguageServer.Testing
     [TestClass]
     public class ProjectLoaderTests
     {
-        [AssemblyInitialize]
-        public static void SetupMSBuildLocator(TestContext testContext)
-        {
-            VisualStudioInstance vsi = MSBuildLocator.RegisterDefaults();
-
-            // This is replicating the approach followed in Microsoft.Quantum.QsLanguageServer.Server.Run()
-            AssemblyLoadContext.Default.Resolving += (assemblyLoadContext, assemblyName) =>
-            {
-                string path = Path.Combine(vsi.MSBuildPath, assemblyName.Name + ".dll");
-                if (File.Exists(path))
-                {
-                    return assemblyLoadContext.LoadFromAssemblyPath(path);
-                }
-
-                return null;
-            };
-        }
+        static ProjectLoaderTests() =>
```

Comment:
Hm, I am not sure this change here will always work correctly - I believe the locator throws if registering defaults is called multiple times or too late. I suggest to keep it as an AssemblyInitialize method, though I may be wrong about the need/benefits.

Reply:
I ran into some cases where `AssemblyInitialize` was not being respected correctly; on the IQ# kernel, isolated the call from unit testing harness to `RegisterDefaults` in a `Lazy<VisualStudioInstance>` property on a static class.

Reply:
I just pushed up a new version. Is that what you meant, @cgranade?